docs/document/content/dev-manual/mask.cn.md

```diff
@@ -26,3 +26,4 @@ chapter = true
 | MASK_FROM_X_TO_Y              | 遮盖自 x 至 y 数据脱敏算法 | [`org.apache.shardingsphere.mask.algorithm.cover.MASK_FROM_X_TO_Y`](https://github.com/apache/shardingsphere/blob/master/features/mask/core/src/main/java/org/apache/shardingsphere/mask/algorithm/cover/MaskFromXToYMaskAlgorithm.java)                 |
 | MASK_BEFORE_SPECIAL_CHARS     | 特殊字符前遮盖数据脱敏算法   | [`org.apache.shardingsphere.mask.algorithm.cover.MASK_BEFORE_SPECIAL_CHARS`](https://github.com/apache/shardingsphere/blob/master/features/mask/core/src/main/java/org/apache/shardingsphere/mask/algorithm/cover/MaskBeforeSpecialCharsAlgorithm.java)                 |
 | MASK_AFTER_SPECIAL_CHARS      | 特殊字符后遮盖数据脱敏算法   | [`org.apache.shardingsphere.mask.algorithm.cover.MASK_AFTER_SPECIAL_CHARS`](https://github.com/apache/shardingsphere/blob/master/features/mask/core/src/main/java/org/apache/shardingsphere/mask/algorithm/cover/MaskAfterSpecialCharsAlgorithm.java)                 |
+| PERSONAL_IDENTITY_NUMBER_RANDOM_REPLACE      | 身份证前6位行政区编码随机替换脱敏算法   | [`org.apache.shardingsphere.mask.algorithm.replace.PERSONAL_IDENTITY_NUMBER_RANDOM_REPLACE`](https://github.com/apache/shardingsphere/blob/master/features/mask/core/src/main/java/org/apache/shardingsphere/mask/algorithm/replace/PersonalIdentityNumberRandomReplaceAlgorithm.java)                 |
```

Comment:
Hi @liushangqing9803, what we need to replace is the birthday information in the middle of the ID card.